src/test/java/bio/terra/service/filedata/azure/AzureSynapsePdaoUnitTest.java

```diff
@@ -21,73 +28,79 @@
 import bio.terra.service.common.AssetUtils;
 import bio.terra.service.dataset.AssetSpecification;
 import bio.terra.service.dataset.exception.TableNotFoundException;
+import bio.terra.service.dataset.flight.ingest.IngestUtils;
+import bio.terra.service.filedata.DrsIdService;
+import bio.terra.service.resourcemanagement.azure.AzureResourceConfiguration;
 import bio.terra.service.snapshot.SnapshotTable;
 import bio.terra.service.snapshot.exception.InvalidSnapshotException;
 import bio.terra.service.tabulardata.WalkRelationship;
+import com.fasterxml.jackson.databind.ObjectMapper;
 import com.microsoft.sqlserver.jdbc.SQLServerException;
-import java.io.IOException;
 import java.sql.SQLException;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 import java.util.UUID;
-import org.junit.Before;
-import org.junit.Test;
-import org.junit.experimental.categories.Category;
-import org.junit.runner.RunWith;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.beans.factory.annotation.Qualifier;
-import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
-import org.springframework.boot.test.context.SpringBootTest;
-import org.springframework.boot.test.mock.mockito.MockBean;
-import org.springframework.boot.test.mock.mockito.SpyBean;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
 import org.springframework.dao.DataAccessException;
 import org.springframework.jdbc.core.namedparam.MapSqlParameterSource;
 import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate;
-import org.springframework.test.context.ActiveProfiles;
-import org.springframework.test.context.junit4.SpringRunner;
 import org.stringtemplate.v4.ST;
 
-@RunWith(SpringRunner.class)
-@SpringBootTest(properties = {"datarepo.testWithEmbeddedDatabase=false"})
-@AutoConfigureMockMvc
-@ActiveProfiles({"google", "unittest"})
-@Category(Unit.class)
-public class AzureSynapsePdaoUnitTest {
-  private AuthenticatedUserRequest testUser =
-      AuthenticatedUserRequest.builder()
-          .setSubjectId("DatasetUnit")
-          .setEmail("dataset@unit.com")
-          .setToken("token")
-          .build();
+@ExtendWith(MockitoExtension.class)
+@Tag(Unit.TAG)
+class AzureSynapsePdaoUnitTest {
+  private static final AuthenticatedUserRequest TEST_USER =
+      AuthenticationFixtures.randomUserRequest();
   private SnapshotRequestAssetModel requestAssetModel;
   private AssetSpecification assetSpec;
   private WalkRelationship walkRelationship;
 
-  @Autowired AssetUtils assetUtils;
-  @SpyBean AzureSynapsePdao azureSynapsePdaoSpy;
```

Comment:
A few tests need to `spy()` this class so they can intercept calls to `executeSynapseQuery()`. Instead of spying generally, a spy is created in those specific tests that need it. Adding directly mockable interface layer for query execution might be better approach but would require changes to the class being tested.